lib/config/presets/internal/replacements.ts

```diff
@@ -38,6 +39,54 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  'emotion-10-to-11': {
+    description:
+      'various unscoped emotion v10 packages became renamed & scoped in v11',
+    packageRules: [
+      {
+        matchCurrentVersion: '>=11.0.0',
```

Comment:
```suggestion
        matchCurrentVersion: '^10.0.0',
```

Shouldn't they all be like this? Ie replace if current version is 10?

Reply:
Perhaps. I was working under the assumption that the replacement is only useful once they choose to adopt V11. If we use `^10.0.0` then wouldn't that be detrimental to any rules that use `allowedVersions: '10'` with the intention of allowing minor/patch upgrades within the 10.x range without committing to a 11.x upgrade?

As an aside, the docs all show `>=` syntax. Is the caret the new preferred approach? :)

Reply:
I think `>=` is ok, we use it in other rules too.

Reply:
Sorry, in the case you're right. I thought there was a hard switchover from 10->11